src/EditorFeatures/Core/Implementation/CodeCleanup/CodeCleanupService.cs

```diff
@@ -0,0 +1,165 @@
+            dictionary.Add(FeatureOnOffOptions.FixThisQualification,
+                new[] { IDEDiagnosticIds.AddQualificationDiagnosticId, IDEDiagnosticIds.RemoveQualificationDiagnosticId });
+            dictionary.Add(FeatureOnOffOptions.FixFrameworkTypes,
+                new[] { IDEDiagnosticIds.PreferFrameworkTypeInDeclarationsDiagnosticId, IDEDiagnosticIds.PreferFrameworkTypeInMemberAccessDiagnosticId });
+            dictionary.Add(FeatureOnOffOptions.FixAddRemoveBraces,
+                new[] { IDEDiagnosticIds.AddBracesDiagnosticId });
+            dictionary.Add(FeatureOnOffOptions.FixAccessibilityModifiers,
+                new[] { IDEDiagnosticIds.AddAccessibilityModifiersDiagnosticId });
+            dictionary.Add(FeatureOnOffOptions.SortAccessibilityModifiers,
+                new[] { IDEDiagnosticIds.OrderModifiersDiagnosticId });
+            dictionary.Add(FeatureOnOffOptions.MakeReadonly,
+                new[] { IDEDiagnosticIds.MakeFieldReadonlyDiagnosticId });
+            dictionary.Add(FeatureOnOffOptions.RemoveUnnecessaryCasts,
+                new[] { IDEDiagnosticIds.RemoveUnnecessaryCastDiagnosticId });
+            dictionary.Add(FeatureOnOffOptions.FixExpressionBodiedMembers,
+                new[] { IDEDiagnosticIds.UseExpressionBodyForConstructorsDiagnosticId,
+                IDEDiagnosticIds.UseExpressionBodyForMethodsDiagnosticId,
+                IDEDiagnosticIds.UseExpressionBodyForConversionOperatorsDiagnosticId,
+                IDEDiagnosticIds.UseExpressionBodyForOperatorsDiagnosticId,
+                IDEDiagnosticIds.UseExpressionBodyForPropertiesDiagnosticId,
+                IDEDiagnosticIds.UseExpressionBodyForIndexersDiagnosticId,
+                IDEDiagnosticIds.UseExpressionBodyForAccessorsDiagnosticId });
+            dictionary.Add(FeatureOnOffOptions.FixInlineVariableDeclarations,
+                new[] { IDEDiagnosticIds.InlineDeclarationDiagnosticId });
+            dictionary.Add(FeatureOnOffOptions.RemoveUnusedVariables,
+                new[] { "CS0168", "CS0219" });
+            dictionary.Add(FeatureOnOffOptions.FixObjectCollectionInitialization,
+                new[] { IDEDiagnosticIds.UseObjectInitializerDiagnosticId, IDEDiagnosticIds.UseCollectionInitializerDiagnosticId });
+            //dictionary.Add(FeatureOnOffOptions.FixLanguageFeatures,
+            //    new[] { IDEDiagnosticIds. });
+            //IDEDiagnosticIds.PreferIntrinsicPredefinedTypeInDeclarationsDiagnosticId,
+            //    IDEDiagnosticIds.PreferIntrinsicPredefinedTypeInMemberAccessDiagnosticId,
+            //    IDEDiagnosticIds.InlineAsTypeCheckId,
+            //    IDEDiagnosticIds.InlineIsTypeCheckId,
+            //    IDEDiagnosticIds.InlineIsTypeWithoutNameCheckDiagnosticsId,
+
+            return dictionary.ToImmutableDictionary();
+        }
+
+        public Task<IEnumerable<TextChange>> GetChangesForCleanupDocument(Document document, CancellationToken cancellationToken)
```

Comment:
feels weird for this to return text changes.  Why not have it just return the new document?

Reply:
@heejaechang could you answer this?

Reply:
fixed